src/setupBrowser.ts

```diff
@@ -48,3 +48,39 @@ export function setupBrowser() {
     })
   }
 }
+
+function disableLocalStorage() {
```

Comment:
I think we also need to disable it for games running inside our app? If so, we might want to move it into another file for clarify :)